lib/haxe/src/org/apache/thrift/transport/TFileStream.hx

```diff
@@ -24,6 +24,9 @@ import haxe.io.BytesBuffer;
 import haxe.io.Input;
 import haxe.io.Output;
 
+#if js
+import js.node.Fs;
+#end
 
```

Comment:
as for me current implementation is better over scattering the same functionality over different files 
i.e. always visible 'original' implementation as well js part which just reimplements original one in target depended way